bumpversion/cli.py

```diff
@@ -279,14 +279,14 @@ def _load_configuration(config_file, explicit_config, defaults):
 
     part_configs = {}
     files = []
-    file_or_part = re.compile("^bumpversion:(file|part):(.+)")
+    file_or_part = re.compile("^bumpversion:(file(_[^:]+)?|part):(.+)")
```

Comment:
Would it be easier to use named groups instead? Or just split on `:`

Reply:
Using named groups in a later update.